docs/src/modules/components/TopLayoutBlog.js

```diff
@@ -161,9 +161,12 @@ const Root = styled('div')(
         lineHeight: 1.7,
       },
       '& img, & video': {
-        borderRadius: 4,
+        border: '1px solid',
+        borderColor: (theme.vars || theme).palette.grey[200],
+        borderRadius: 12,
```

Comment:
A note to acknowledge the impact of this style on other blog posts, e.g. https://master--material-ui.netlify.app/blog/mui-x-end-v6-features/. There are two cases where it can feel a bit weird:

- The image already have a border, it conflicts
- The image already has internal borders, it feels heavy

I don't have a specific preference on how to move forward:

1. Remove the border with inline style for the cases where it's wrong
2. Add the border on an ad-hoc basis

I guess 1. makes sense as it allows to more easily update the whole docs feel. To keep an eye on either 2. is a better default or not.

Reply:
Yeah, you got it, I'm keeping track of these cases! I'll soon put up a PR doing a general image review on all blog posts (including images from the posts themselves as well as standardizing the open graph card image that has since grown inconsistent over the many design iterations I played around with). 😬